Accompany/Views/AddNoteCellTwoTableViewCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AddNoteCellTwoTableViewCellTableViewCell.swift
+//  Accompany
+//
+//  Created by Kelbin David on 2022-02-10.
+//
+
+import UIKit
+import SnapKit
+
+class AddNoteCellTwoTableViewCell: UITableViewCell {
```

Comment:
Please refactor the name of this class and file to "TodoNotesTableViewCell", it's more semantic and readable.
```suggestion
class TodoNotesTableViewCell: UITableViewCell {
```